mmock.go

```diff
@@ -75,8 +75,14 @@ func getRouter(mocks []definition.Mock, dUpdates chan []definition.Mock) *route.
 }
 
 func loadVarsProcessorEngines(persistPath string) *persist.PersistEngineBag {
-	filePersist := persist.FilePersister{PersistPath: persistPath}
-	persistBag := persist.GetNewPersistEngineBag(filePersist)
+	var persister persist.EntityPersister
+	if strings.Index(persistPath, "mongodb://") < 0 {
```

Comment:
It's ok now, but I want change it in a near future. We can add both engines (or more ) inside the EngineBag and let the user choose which want in the mock configuration through the engine field (already available) What do you think about it? Does it make sense for you? (I'm not sure about if it worth)